server/src/main/java/org/apache/uniffle/server/ShuffleServer.java

```diff
@@ -262,7 +274,7 @@ public StorageManager getStorageManager() {
   }
 
   public Set<String> getTags() {
-    return tags;
+    return Collections.unmodifiableSet(tags);
```

Comment:
Why do we need `Collections.unmodifiableSet`?

Reply:
I think this method is to return read -only tags.